### ISSUE TITLE
Possible source of ssh key issue.

### ISSUE THREAD

**Issue by tomrule007** [garageScript/myProxy#390]
I believe this if statement is always failing because of the quotes around the file name. I'm still a noob with bash so i very well could be missing something but thought i would pass it along.

tested in the terminal with:
```
[ -f  "~/.ssh/authorized_keys" ] || echo "fail"
[ -f ~/.ssh/authorized_keys ] && echo "success"
```

https://github.com/garageScript/myProxy/blob/022ace46b2014e7a8ebdb89c14538b11a911f09f/scripts/setup.sh#L64-L72

**Comment by ggwadera** [comment]
Nice catch, I thinkg the `~` is not expanded because of the quotes, should've been `[ -f "$HOME/.ssh/authorized_keys" ]` to expand into the correct path.

Either way this might not be the issue since this script only runs when first starting the server. After that the process is handled by PM2.

https://github.com/garageScript/myProxy/blob/022ace46b2014e7a8ebdb89c14538b11a911f09f/package.json#L20

**Comment by SlyBouhafs** [comment]
Possible fix with #406
